ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__bundle.yaml

```diff
@@ -0,0 +1,24 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+operator:
```

Comment:
Shouldn't this entry be deleted from the `ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml` file?

Reply:
No we need to build a separate bundle for use outside of the CI.